encoding/gproperties/gproperties.go

```diff
@@ -0,0 +1,176 @@
+// Copyright GoFrame Author(https://goframe.org). All Rights Reserved.
```

Comment:
Why do not you use third party package like [https://github.com/magiconair/properties](https://github.com/magiconair/properties) ?

Reply:
use viper as the decoder/coder lib

Reply:
I found the `viper` is using https://github.com/magiconair/properties as its parser for Java property files.
Is it better just import single package `github.com/magiconair/properties` instead of big package `viper`?

Reply:
Use magiconair as the paser for java properties file.